numpyro/contrib/einstein/steinvi.py

```diff
@@ -25,7 +26,7 @@
 from numpyro.infer.autoguide import AutoGuide
 from numpyro.infer.util import _guess_max_plate_nesting, transform_fn
 from numpyro.optim import _NumPyroOptim
-from numpyro.util import fori_collect, ravel_pytree
+from numpyro.util import fori_collect
```

Comment:
@OlaRonning Does this change look reasonable to you?

Reply:
This was to pass lint - ravel_pytree would no longer be used within numpyro/util.py

Reply:
I checked with a few of my local examples. This change is fine. Thanks @andrewdipper!